setup.json

```
@@ -18,12 +18,16 @@
             "ase.ase = aiida_ase.calculations.ase:AseCalculation"
         ],
         "aiida.parsers": [
-            "ase.ase = aiida_ase.parsers.ase:AseParser"
+            "ase.ase = aiida_ase.parsers.ase:AseParser",
+            "ase.gpaw = aiida_ase.parsers.ase:GpawParser"
+        ],
+        "aiida.workflows": [
+            "ase.gpaw.base = aiida_ase.workflows.base:GpawBaseWorkChain"
         ]
     },
     "python_requires": ">=3.7",
     "install_requires": [
-        "aiida-core[atomic_tools]~=1.0"
+        "aiida-core[atomic_tools]==1.6"
```

You should use the `~=` operator as it will allow higher versions as well and you don't want to restrict it to a minor version. Is there a question you need `1.6` as a lower limit though? Is there any reason 1.5 for example is not compatible?
```suggestion
        "aiida-core[atomic_tools]~=1.6"
```

---

Okay the tests pass with this commit. Have never tested it with a lower version, I think it was initially chosen by what I was using for the `common-workflows` [project](https://github.com/aiidateam/acwf-verification-scripts), is there any reason that the version recommendation there is fixed to `1.6.3`?